docs/user-guide/DREAM-SANS-methodology.ipynb

```diff
@@ -0,0 +1,56 @@
+    "\n",
+    "The general data reduction scheme presented in \"SANS Polarization Analysis Methodology\" will be adapted for the SANS option at the DREAM instrument at the ESS. Following considerations have to be changed with respect to the general methodology:\n",
+    "\n",
+    "### Polarizer:<br>\n",
+    "The DREAM-polarizer is a movable in-situ SEOP <sup>3</sup>He-cell of which the neutron polarization $P^{^3\\textup{He, p}}$ has to be pre-characterized in commissioning measurements and inserted into this workflow. Its wavelength-independent opacity $O^{\\textup{0, p}}$, empty-glass transmission $T^{\\textup{g, p}}$, pre-factor of nuclear polarization $C^{\\textup{p}}$, and its time-decay constant $T^{\\textup{1, p}}$ can be measured once in every commissioning time via Workflow 2 of \"SANS Polarization Analysis Methodology\". The resulting $P^{^3\\textup{He, p}}$ can be inserted into the workflow as constant parameter. To account for sudden fluctuations of $P^{^3\\textup{He, p}}$, which can occur due to changes in pressure or the like, the beam intensity after the polarizer has to be continuously recorded as function of time. Sudden fluctuations in intensity above a pre-defined difference in time must pop out an error message \"Critical intensity fluctuations detected in polarizer-monitor. Re-measure $P^{^3\\textup{He, p}}$!\".<br>\n",
+    "Note: Workflow 1 of \"SANS Polarization Analysis Methodology\" cannot be used for this polarizer. \n",
+    "\n",
+    "### Analyzer:<br>\n",
+    "In the current scope, DREAM will be equipped with 3 different SANS analyzers: (i) an ex-situ SEOP <sup>3</sup>He-cell, (ii) an in-situ SEOP <sup>3</sup>He-cell, and (iii) a SANS-window being part of a wide angle MEOP based <sup>3</sup>He-cell.\n",
+    "\n",
+    "(i)  The ex-situ SEOP <sup>3</sup>He-cell is a movable, time-dependent cell and has to be treated as described in \"SANS Polarization Analysis Methodology\".<br>\n",
+    "(ii)  The in-situ SEOP <sup>3</sup>He-cell is a movable, non-time-dependent cell and has to be treated as described above for the DREAM polarizer.<br>\n",
+    "(iii)  The ex-situ MEOP wide angle <sup>3</sup>He-cell with a small SANS window is a rotatable, time-dependent cell and has to be treated as the analyzer described in \"DREAM-diffraction methodology\".\n",
+    "\n",
+    "### Monitors:<br>\n",
+    "For an optimal SANS data-reduction, three monitors would be ideal: (i) an incident beam monitor $M_{\\textup{in}}$ before the polarizer, (ii) a monitor after the polarizer $M_{\\textup{p}}$, and (iii) a monitor after the analyzer $M_{\\textup{a}}$ (alternatively on the main detector if the direct beam can be recorded). At DREAM, only two monitors will be inserted: $M_{\\textup{p}}$ and $M_{\\textup{a}}$. In general, the total beam intensity is corrected by $M_{\\textup{in}}$. Due to the time-independence of $P^{^3\\textup{He, p}}$, the beam intensity of $M_{\\textup{p}}$ can be used instead.\n",
```

Comment:
> At DREAM, only two monitors will be inserted: $M_{\\textup{p}}$ and $M_{\\textup{a}}$.

Are there plans to maybe use one of the mobile monitors that is able to be shared between instruments for $M_{\rm{in}}$?